assignments/callbacks.js

```diff
@@ -21,32 +21,63 @@ const items = ['Pencil', 'Notebook', 'yo-yo', 'Gum'];
 
 */
 
-
 function getLength(arr, cb) {
+  
+  return cb(arr.length)
+
   // getLength passes the length of the array into the callback.
 }
 
 function last(arr, cb) {
+    // items[0] ; how to get the first item
+     return cb(arr[arr.length -1])
```

Comment:
this won't work because you haven't defined a callback. Do something like - . 

```javascript
function logToConsole(item) {
	console.log(item);
}

getLength(items, logToConsole);
```